### PR TITLE
Ref

### PR DESCRIPTION
**Pull request by teonlamont**
Fix reference in io manual.

Add note to KIT readers.

### CODE REVIEW

File: mne/io/kit/kit.py [hunk 1]
```diff
@@ -72,6 +72,13 @@ class RawKIT(_BaseRaw):
     verbose : bool, str, int, or None
         If not None, override default verbose level (see mne.verbose).
 
+    Notes
+    -----
+    elp and hsp are usually the exported text files (*.txt) from the Polhemus
```
Comment (Eric89GXL) [inline mne/io/kit/kit.py]: might as well make these render nicely as code with double-backticks around var names

**Comment by Eric89GXL** [comment]
Other than my minor gripe, LGTM with my limited knowledge of that system

File: doc/manual/io.rst [hunk 2]
```diff
@@ -167,10 +167,10 @@ This operation is performed by :ref:`mne_compensate_data`.
 Importing KIT MEG system data
 =============================
 
-MNE-Python includes the :func:`mne.io.read_raw_kit` and
+MNE-Python includes the :func:`mne.read_raw_kit` and
```
Comment (agramfort) [inline doc/manual/io.rst]: the correct path is : `mne.io.read_raw_kit`

I don't get this change

Comment (teonlamont) [inline doc/manual/io.rst]: accident, change to the wrong function

**Comment by agramfort** [comment]
thanks @teonlamont

**Pull request merged**